### PR TITLE
Rm xstream annotations and converters from provisioning classes

### PR DESCRIPTION
**Pull request by erikrozendaal**
Based on PR #41

### CODE REVIEW

File: src/main/java/net/ripe/rpki/commons/provisioning/payload/issue/response/CertificateIssuanceResponsePayloadSerializer.java [hunk 1]
```diff
@@ -29,19 +29,36 @@
...
+public class CertificateIssuanceResponsePayloadSerializer extends AbstractProvisioningPayloadXmlSerializer<CertificateIssuanceResponsePayload> {
+    public CertificateIssuanceResponsePayloadSerializer() {
+        super(PayloadMessageType.issue_response);
     }
 
-    @Override
-    public XStreamXmlSerializer<CertificateIssuanceResponsePayload> build() {
-        getXStream().processAnnotations(CertificateIssuanceResponsePayload.class);
-        return super.build();
+    protected CertificateIssuanceResponsePayload parseXmlPayload(Element messageElement) throws IOException {
```
Comment (wibisono) [inline src/main/java/net/ripe/rpki/commons/provisioning/payload/issue/response/CertificateIssuanceResponsePayloadSerializer.java:50]: Idea seems to think this is not throwing IOException. (I also can only find DOMException/DOMSerializerException)

Comment (lolepezy) [reply src/main/java/net/ripe/rpki/commons/provisioning/payload/issue/response/CertificateIssuanceResponsePayloadSerializer.java:50]: I guess, it's coming from the base class?

**Review by lolepezy — APPROVED** [review]
I am generally fine with it, but we need to have an explicit testing phase for it.

File: src/main/java/net/ripe/rpki/commons/provisioning/payload/issue/response/CertificateIssuanceResponsePayloadSerializer.java (hunk 1 above)
Comment (wibisono) [reply src/main/java/net/ripe/rpki/commons/provisioning/payload/issue/response/CertificateIssuanceResponsePayloadSerializer.java:50]: Could be that it was, but that @Override is no longer there.

**Pull request merged**